packages/npm-dependencies-validation/test/utils/npmUtil.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { spawnCommand, getNPM } from "../../src/utils/npmUtil";
+import { expect } from "chai";
+
+describe("npmUtil unit test", () => {
+  it("spawnCommand failed", () => {
+    return expect(
+      spawnCommand(["ls"], "./non_existing_path")
+    ).to.be.rejectedWith(`spawn ${getNPM()} ENOENT`);
+  });
+
+  it("spawnCommand succeeded with ls", () => {
+    return expect(spawnCommand(["ls", "--depth=0"], "./")).to.be.fulfilled;
+  });
+
+  describe("npm command", () => {
+    const originalPlatform = process.platform;
+
+    after(function () {
```

Comment:
after**Each**

Reply:
No, I wanted it to set original value only after all tests in this describe

Reply:
this damages tests isolation.
the Nth test has a value from the Nth-1 test...
So execution could be different, e.g if we run all the tests in sequence or if we run only one test.
(I wasted hours on a similar problem in this very same repo...)